vault/logical_system_quotas.go

```diff
@@ -301,7 +301,10 @@ func (b *SystemBackend) handleRateLimitQuotasUpdate() framework.OperationFunc {
 		}
 
 		mountPath := sanitizePath(d.Get("path").(string))
-		ns := namespace.RootNamespace
+		ns, err := namespace.FromContext(ctx)
+		if err != nil {
+			return logical.ErrorResponse("invalid namespace"), nil
```

Comment:
By doing a exact string matching against the repo, there are 6 examples of following error format when we cannot retrieve the namespace from the context, so I'd suggest using something like that
```suggestion
			return logical.ErrorResponse(fmt.Errorf("failed to find namespace in context: %w", err).Error()), nil
```